### ISSUE TITLE
Unable to install/update jellyfin

### ISSUE THREAD

**Issue by dziurka** [YunoHost-Apps/jellyfin_ynh#148]
It appears that jellyfin has removed an older version of the installer from its servers (v 10.8.11-1), making it impossible to update the application.

```bash
Error: Provisioning, deprovisioning, or updating resources for jellyfin failed: Failed to download asset 'server' (https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb) for jellyfin: Will not apply HSTS. The HSTS database must be a regular and non-world-writable file.
ERROR: could not open HSTS store at '/root/.wget-hsts'. HSTS will be disabled.
https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb:
2024-02-19 17:36:36 ERROR 404: Not Found.
```

https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb returns 404

![image](https://github.com/YunoHost-Apps/jellyfin_ynh/assets/16003856/a84882a4-7423-4445-955b-3181719c4c5a)

**Comment by Psyquoquack** [comment]
Same here.
Tested the testing branch: 
Resolved the issue but brings an other about jellyfin-ffmpeg. [https://paste.yunohost.org/raw/wikenazure](https://paste.yunohost.org/raw/wikenazure)

**Comment by Jefferson-tan** [comment]
I have the same issue as well. [Here](https://paste.yunohost.org/raw/usumupatam) are my logs:
```
args:
  app: jellyfin
  force: false
  label: undefined
  no_remove_on_failure: false
ended_at: 2024-02-20 13:25:25.137289
error: 'Failed to download asset ''server'' (https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb)
  for jellyfin: https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb:

  2024-02-20 20:25:25 ERROR 404: Not Found.

  '
interface: api
operation: app_install
parent: null
related_to:
- - app
  - jellyfin
started_at: 2024-02-20 13:25:23.997293
success: false
yunohost_version: 11.2.10

============

2024-02-20 20:25:24,002: INFO - Installing jellyfin...
2024-02-20 20:25:24,006: INFO - Provisioning sources...
2024-02-20 20:25:24,010: DEBUG - Prefetching asset server: https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb ...
2024-02-20 20:25:25,133: WARNING - Failed to provision sources : Failed to download asset 'server' (https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb) for jellyfin: https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb:
2024-02-20 20:25:25 ERROR 404: Not Found.

2024-02-20 20:25:25,135: INFO - Deprovisioning sources...
2024-02-20 20:25:25,136: ERROR - Provisioning, deprovisioning, or updating resources for jellyfin failed: Failed to download asset 'server' (https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb) for jellyfin: https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb:
2024-02-20 20:25:25 ERROR 404: Not Found.
```

**Comment by Psyquoquack** [comment]
Just made a temporary fix on my fork only for amd64
I worked fine on mine. Can you try it @dziurka @Jefferson-tan ??? Thanks ;)

**Comment by Jefferson-tan** [comment]
I made a temporary fix on my fork myself, but I ran into some issue at the end of the installation process
https://paste.yunohost.org/raw/eciwefohoh
```
2024-02-20 21:30:46,270: WARNING - touch: cannot touch '/etc/jellyfin/system.xml': No such file or directory
2024-02-20 21:30:46,270: DEBUG - + ynh_exit_properly
2024-02-20 21:30:46,270: DEBUG - + [[ install =~ ^install$|^upgrade$|^restore$ ]]
2024-02-20 21:30:47,782: ERROR - Unable to install jellyfin: An error occurred inside the app installation script
```

> Same here. Tested the testing branch: Resolved the issue but brings an other about jellyfin-ffmpeg. https://paste.yunohost.org/raw/wikenazure

I think I know why, they used FFmpeg version `6.0.1-1` in the testing branch but that version no longer exists on Jellyfin's repos.

**Comment by Jefferson-tan** [comment]
> Just made a temporary fix on my fork only for amd64 I worked fine on mine. Can you try it @dziurka @Jefferson-tan ??? Thanks ;) I might be a vilain, verify my code before !

I've just tested your fork and it doesn't work, looks like you haven't updated the `manifest.toml` file with the updated links yet.

```
2024-02-20 22:06:50,526: WARNING - Failed to provision sources : Failed to download asset 'server' (https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb) for jellyfin: https://repo.jellyfin.org/releases/server/debian/versions/stable/server/10.8.11/jellyfin-server_10.8.11-1_amd64.deb:
2024-02-20 22:06:50 ERROR 404: Not Found.
```

**Comment by Psyquoquack** [comment]
Sorry, I forgot to mention that i made my fix on the testing branch of my repo.
To resolve the error you had, I mkdir the /etc/jellyfin folder then deleted it. I don't know why, but it worked after...

**Comment by Jefferson-tan** [comment]
> Sorry, I forgot to mention that i made my fix on the testing branch of my repo.
Just took a quick look, seems like we have essentially the same code now.

> To resolve the error you had, I mkdir the /etc/jellyfin folder then deleted it. I don't know why, but it worked after...
That's odd..

**Comment by Psyquoquack** [comment]
This is my history of commands:
```
   85  cd /etc/
   86  mkdir jellyfin
   87  sudo mkdir jellyfin
   88  sudo yunohost app install https://github.com/Psyquoquack/jellyfin_ynh/tree/testing
   89  sudo rm -r /etc/jellyfin
   90  sudo yunohost app install https://github.com/Psyquoquack/jellyfin_ynh/tree/testing
```

**Comment by Jefferson-tan** [comment]
Just tried that, Jellyfin is up and running now

**Comment by ahstro** [comment]
This should be fixed now, with version `10.8.13` being live.

**Comment by gguzzina** [comment]
This seems to be happening again, due to Jellyfin shuffling around the folder structure of their repo. A PR to fix this exists.

**Comment by cichy1173** [comment]
Yes, I tried to reinstall Jellyfin due to problems with encoding and I cannot install it again https://paste.yunohost.org/raw/elelahamuq

**Comment by csolisr** [comment]
That reminds me, is there some technical reason why we can't bump the version to 10.9.x? Especially considering that the Android clients are already warning that they'll drop support for 10.8.x in the next version. Is there a dependence unavailable on Bullseye?

**Comment by pagiraud** [comment]
@csolisr See #154

**Comment by kay0u** [comment]
fixed by #158
